code/modules/mining/boulder_processing/boulder.dm

```diff
@@ -22,6 +22,8 @@
 	var/boulder_size = BOULDER_SIZE_SMALL
 	/// Used in inheriting the icon_state from our parent vent in update_icon.
 	var/boulder_string = "boulder"
+	// There is one boulder per boulder (this is required for the Clarke UI as it treats ores and boulders in the same fashion and needs an amount for both)
+	var/amount = 1
```

Comment:
Fuck, I was under the impression the boulder was a subtype of ore for a second and merged it. This is not a long-lasting way to fix the issue, and it'll lead to some technical debt eventually.
Basically, don't make a new variable with a similar name to fix an issue that's actually about an item that is not being an ore stack being inside a loop that's meant for ore stacks!